tests/integration/actions/config/base.py

```diff
@@ -74,11 +77,22 @@ def test(self, request, tmux_session, step):
 
         if step.mask:
             # mask out some configuration that is subject to change each run
-            mask = "X" * 50
-            maskables = ["BECOME_PLUGIN_PATH", "CACHE_PLUGIN_CONNECTION", "COLLECTIONS_PATHS"]
-            for idx, line in enumerate(received_output):
-                if any(m in line for m in maskables):
-                    received_output[idx] = mask
+            maskables = [
+                "BECOME_PLUGIN_PATH",
+                "CACHE_PLUGIN_CONNECTION",
+                "COLLECTIONS_PATHS",
+                "DEFAULT_CALLBACK_PLUGIN_PATH",
+                "DEFAULT_LOCAL_TMP",
+            ]
+            # Determine if a menu is showing
+            mask_column_name = "CURRENT VALUE"
+            column_start = received_output[0].find(mask_column_name)
+            column_exists = column_start != -1
+            if column_exists:
+                mask = len(mask_column_name) * "X"
+                for idx, line in enumerate(received_output):
+                    if any(f"│{m}" in line for m in maskables):
+                        received_output[idx] = received_output[idx][0:column_start] + mask
```

Comment:
Since the previous conversation got hidden — could you explain why it is necessary to cut the end of the line together with the masked thing?

Reply:
I think I answered this above, at least what I think you are asking:

The length of the value to mask can change based on the developers workstation, so I used the minimum possible column width which is always the length of the column header. If we masked the the actual of the value with X * length of the value, we might still changes between developer system where the fixtures were generated.